RunGcm-filters-leticia.ipynb

```diff
@@ -0,0 +1,3088 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ssh_filtered = filter_cpu.apply(ssh, dims=['nlat', 'nlon'])
</pre><p>You could add a line</p><pre class="ql-syntax" spellcheck="false">ssh_filtered  # note that this is a dask array, so we have filtered "lazily"
</pre><p>which would show that we still have a dask array, which means that we have filtered "lazily" so far, i.e., nothing has actually been computed yet. Dask has just built up the task graph!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oceanhackweek/ohw22-proj-gcmfilters/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>